js/graphs.js

```diff
@@ -434,39 +452,43 @@ function createGraph(graph, data) {
-
+	
 	var baseTitle = graph.properties.title;
 
-	if (graph.blocksConfig && typeof (graph.blocksConfig['title']) !== 'undefined') {
+	if (graph.blocksConfig && isDefined(graph.blocksConfig['title'])) {
 		baseTitle = graph.blocksConfig['title'];
 	}
-	
-	var iconColour = graph.blocksConfig && isDefined(graph.blocksConfig['iconColour'])? graph.blocksConfig['iconColour'] : '#ccc';
 
-	var title = '<div class="graphheader"><div class="graphtitle"><i class="fas fa-chart-bar" style="font-size:20px;margin-left:5px;color:' + iconColour + '">&nbsp;</i>' + baseTitle;
-	if(!graph.multigraph){
+	var title = '<div class="graphheader"><div class="graphtitle"><i class="fas fa-chart-bar" style="font-size:20px;margin-left:5px;color:' + buttonIcon + '">&nbsp;</i>' + baseTitle;
+	//if (typeof (graph.properties.currentValue) !== 'undefined' && graph.properties.currentValue !== 'undefined') title += ': <B class="graphcurrent' + graph.properties.idx + '">' + graph.properties.currentValue + ' ' + graph.properties.txtUnit + '</B>'; // <--why this row?
+	var span = '&nbsp;<span class="graphcurrent' + graph.properties.idx + '">&nbsp;<i class="fas fa-equals" style="font-size:14px;color:' + buttonIcon + '">&nbsp;</i>&nbsp;';
+	
+	if(!graph.multigraph ){
 		if(isDefined(graph.properties.currentValue)) {
-			title += '&nbsp;<span class="graphcurrent' + graph.properties.idx + '">&nbsp;<i class="fas fa-equals" style="font-size:14px;color:' + iconColour + '">&nbsp;</i>&nbsp;' + graph.properties.currentValue + ' ' + graph.properties.txtUnit + '</span>';
+			title += span + graph.properties.currentValue + ' ' + graph.properties.txtUnit + '</span>';
 		}
 	} else {
 		if (isDefined(graph.properties.currentValues)) {
-			title += '&nbsp;<span class="graphcurrent' + graph.properties.idx + '">&nbsp;<i class="fas fa-equals" style="font-size:14px;color:' + iconColour + '">&nbsp;</i>&nbsp;' + graph.properties.currentValues.join('<span style="color:' + iconColour + ';font-weight:900;font-size:16px;"> | </span>') + ' ' + graph.properties.txtUnit + '</span>';
+			title += span + graph.properties.currentValues.join('<span style="color:' + graph.properties.buttons.icon + ';font-weight:900;font-size:16px;"> | </span>') + ' ' + graph.properties.txtUnit + '</span>';
 		}
 	}
 	title += '</div>';
 
 	var width = 12;
-	if (graph.blocksConfig && typeof (graph.blocksConfig['width']) !== 'undefined' && !graph.properties.popup) {
+	if (graph.blocksConfig && isDefined(graph.blocksConfig['width']) && !graph.properties.popup) {
 		width = graph.blocksConfig['width'];
 	}
 	var html = '';
 	html += title + '<div class="graphbuttons" >' + buttons + '</div>';
 	html += '</div>'
 
-	html += '<div class="graph swiper-no-swiping' + (graph.properties.popup ? ' popup graphheight' : '') + '" id="graph' + graph.properties.idx + '">';
+	html += '<div class="graph swiper-slide' + (graph.properties.popup ? ' popup graphheight' : '') + '" id="graph' + graph.properties.idx + '">';
```

Comment:
Adding class swiper-slide is not correct: It kills the graph height. The class swiper-no-swiping prevents to swipe to a next screen, if you swipe along the graph line on touch devices. I propose to undo the change of this line.

Reply:
OK. Understood. The reason for updating it to '-slide' is because screens don't slide when you have multiple graphs filing the whole screen. The only way to get back to the previous screen (using touch or mouse) is to "try" and click the tiny margin in between the graphs.

Reply:
ok. The swiper-no-swiping is not really needed. Then let's just remove swiper-slide. ok?

Reply:
Great compromise 👍 Thanks.